packages/conversation/src/rest/v1/conversation-domain-api.ts

```diff
@@ -59,9 +61,13 @@ export class ConversationDomainApi implements Api {
    */
   public getSinchClient(): ApiClient {
     if (!this.client) {
+      const region = this.sinchClientParameters.region || Region.UNITED_STATES;
+      if(!Object.values(ConversationRegion).includes((region as unknown) as ConversationRegion)) {
+        throw new Error(`The region '${region}' is not supported for the Conversation API`);
```

Comment:
What about only logging an error/warning?
Imagine a new region is supported next months but SDK is not yet ready nor released. 
An end user wish to test the new region (it was one of it request to support) but he won't be able to. Worst if the region is in beta test it mean we have to release a new version to test a alpha/beta feature just due to a config level setting.

Reply:
Thanks for your comment. I've updated the code and the test accordingly